tests/qos/test_qos_sai.py

```diff
@@ -1617,6 +1617,12 @@ def testQosSaiPgSharedWatermark(
             pktsNumFillShared = int(
                 qosConfig[pgProfile]["pkts_num_trig_egr_drp"]) - 1
 
+        if dutTestParams["basicParams"].get("platform_asic", None) \
+                == "cisco-8000":
+            if not get_src_dst_asic_and_duts['single_asic_test']:
+                if pgProfile == "wm_pg_shared_lossy":
+                    pytest.skip("The lossy test is not valid for multiAsic configuration.")
```

Comment:
what's the reason we can not support lossy test in chassis?

Reply:
@kevinskwang : The PGShared watermark is the property of incoming port. In case of mAsic or chassis, the incoming port is actually the backplane port. Since we don't have infra in sonic-mgmt to find the correct backplane port that traffic is acutally using, we can't reliably do this test. Also this is covered in singleAsic tests.